paramiko/pkey.py

```diff
@@ -43,6 +43,16 @@
 from paramiko.message import Message
 
 
+# The location of the TripleDES Cipher is changing from cryptography>=43.0.0
+# and scheduled for removal in cryptography==48.0.0. This is temporary change
```

Comment:
The TripleDES cipher is not being removed from the cryptography library. It will just be removed from the `cryptography.hazmat.primitives.ciphers.algorithms` module. So I don't think this is a temporary change.

An update to the deprecation message has been merged upstream. Which hopefully will make it clearer.

https://github.com/pyca/cryptography/pull/11407

Reply:
@JohnVillalovos updated